### PR TITLE
Erin/questionnaire refactoring

### PR DESCRIPTION
**Pull request by erinysong**
## Updated Fields on Personal and Professional Questionnaires

Personal and professional questionnaires now updated with Unloop's updated fields. Database now stores the new fields and the forms on the frontend now reflect these changes. 

TODO [4/26]: 
- making the race/ethnicities field into a dropdown menu
- converting the personal questionnaire into a multi-step form
- ordering questionnaire questions in correct order

### Related PRs

Optional - any related PRs you're waiting on, or PRs that will conflict, etc

### Migrations

Optional - if you added anything to the database through migration(s)

### Screenshots

Optional but would be nice for final presentation :)

CC: @didvi

### CODE REVIEW

**Comment by didvi** [comment]
Can you remove the yarn.lock file from your PR, you don't have some of the dependencies we recently added

**Comment by didvi** [comment]
Looks good! There's a couple small things to fix up but they shouldn't be too much work.
- Make the header for the questionnaire (where it says Personal Information or Professional Information) smaller -- around 24px height with a font size 22.5 (same as updated dashboards)
- can you make the border radius the same as theme.shape.borderRadius
- Doc status should be DOC status (unloop wanted this to be capitalized) and have radio box options: WR, GRE, Community placement, Released with placement, Released without placement, instead of being a textbox
- can you make sure the fields for the questionnaires are in the same order as the google doc
- race and ethnicities should be the select we talked about, not a text field
- course completion should be radio boxes with two options: completed or incomplete

**Comment by didvi** [comment]
also, when I add in a birthdate, it doesn't save if it doesn't have the right format, can you either change this to accept all formats or add some sort of notification (make the box red, there's a tag in material ui for this) so that the user knows that their entry is not in the correct format

**Comment by erinysong** [comment]
thanks for the feedback! working on changes rn and I'll update the list below to indicate what's done so far

- [ ] Make the header for the questionnaire (where it says Personal Information or Professional Information) smaller -- around 24px height with a font size 22.5 (same as updated dashboards)

- [ ] can you make the border radius the same as theme.shape.borderRadius

- [ ] Doc status should be DOC status (unloop wanted this to be capitalized) and have radio box options: WR, GRE, Community placement, Released with placement, Released without placement, instead of being a textbox

- [ ] can you make sure the fields for the questionnaires are in the same order as the google doc

- [ ] race and ethnicities should be the select we talked about, not a text field

- [ ] course completion should be radio boxes with two options: completed or incomplete

File: app/javascript/components/QuestionnaireForm/index.js [hunk 1]
```diff
@@ -68,12 +88,247 @@ class QuestionnaireForm extends React.Component {
     }));
   }
 
+  handlePhoneChange(e) {
+    const { id } = e.target;
+    const { value } = e.target;
+    const regex = /^\s*(?:\+?(\d{1,3}))?[-. (]*(\d{3})[-. )]*(\d{3})[-. ]*(\d{4})(?: *x(\d+))?\s*$/;
+    const isValid = regex.test(value);
+    if (isValid) {
```
Comment (didvi) [inline app/javascript/components/QuestionnaireForm/index.js:96]: can we make the field show an error only when there is something entered inside the text field and it is formatted incorrectly

File: app/javascript/components/QuestionnaireForm/index.js.rej [hunk 2]
```diff
@@ -0,0 +1,47 @@
+diff a/app/javascript/components/QuestionnaireForm/index.js b/app/javascript/components/QuestionnaireForm/index.js	(rejected hunks)
```
Comment (didvi) [inline app/javascript/components/QuestionnaireForm/index.js.rej:1]: is this file necessary?

File: app/javascript/components/QuestionnaireModal/index.js.rej [hunk 3]
```diff
@@ -0,0 +1,7 @@
+diff a/app/javascript/components/QuestionnaireModal/index.js b/app/javascript/components/QuestionnaireModal/index.js	(rejected hunks)
```
Comment (didvi) [inline app/javascript/components/QuestionnaireModal/index.js.rej:1]: this too

Comment (erinysong) [reply app/javascript/components/QuestionnaireModal/index.js.rej]: nope, just removed that and the other .rej file
